website/integrations/services/hashicorp-vault/index.md

```diff
@@ -6,12 +6,13 @@ title: Hashicorp Vault
 
 ## What is Vault
 
-> Secure, store and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets and other sensitive data using a UI, CLI, or HTTP API.
+> HashiCorp Vault is an open-source tool for securely managing and storing sensitive data such as passwords, API keys, and encryption keys. 
 >
 > -- https://vaultproject.io
 
 :::note
-This is based on authentik 2022.2.1 and Vault 1.9.3. Instructions may differ between versions. This guide does not cover vault policies. See https://learn.hashicorp.com/tutorials/vault/oidc-auth?in=vault/auth-methods for a more in depth vault guide
+note to self: is this needed
```

Comment:
Don;t forget to take this note to self out, haha. I think we should leave the original wording as is.